packages/eth/tsconfig.base.json

```diff
@@ -0,0 +1,14 @@
+{
+  "include": [],
+  "compilerOptions": {
+    "target": "ESNext",
```

Comment:
I'm assuming you took notes from the spl package for the TS configs here?
Just want to make sure this is compatible with the client TS configs, since they target a lower version than ESNext.

Reply:
yup